plugins/engines/ospray/ispc/render/SimulationRenderer.ispc

```diff
@@ -780,6 +784,35 @@ inline void computeIndirectShading(DifferentialGeometry& dg,
                     attributes.indirectIntensity);
 }
 
+inline void processShadows(DifferentialGeometry& dg,
+                           varying ScreenSample& sample, const varying Ray& ray,
+                           ShadingAttributes& attributes)
+{
+    for (uniform int i = 0; i < attributes.self->abstract.numLights; ++i)
+    {
+        const uniform Light* uniform light =
+            attributes.self->abstract.lights[i];
+        const vec2f s = make_vec2f(0.5f);
```

Comment:
what's the purpose of this value? comment or better name.

Reply:
replaced by samplingLocation